tdrs-frontend/src/components/SubmissionHistory/SubmissionHistory.test.js

```diff
@@ -339,4 +337,181 @@ describe('SubmissionHistory', () => {
     expect(screen.queryByText('test5.txt')).toBeInTheDocument()
     expect(screen.queryByText('test6.txt')).not.toBeInTheDocument()
   })
+
+  it.each([
```

Comment:
I don't love duplicating the test the way I have. I tried creating a table in the argument of it.each like this: `['Pending','Active Case Data'],['Pending', 'Closed Case Data'],` but that caused errors in the components. Any other suggestions?